runtime/runtime-params-estimator/estimator-warehouse/src/check.rs

```diff
@@ -166,11 +215,11 @@ mod tests {
 
         0003a
         {"computed_in":{"nanos":633,"secs":7},"name":"LogBase","result":{"gas":4000000000.0,"instructions":32000.0,"io_r_bytes":0.0,"io_w_bytes":0.0,"metric":"icount","uncertain_reason":null}}
-        {"computed_in":{"nanos":173,"secs":2},"name":"LogByte","result":{"gas":1006000000.0,"instructions":8048.0,"io_r_bytes":0.0,"io_w_bytes":0.0,"metric":"icount","uncertain_reason":null}}
+        {"computed_in":{"nanos":173,"secs":2},"name":"LogByte","result":{"gas":1006000000.0,"instructions":8048.0,"io_r_bytes":0.0,"io_w_bytes":0.0,"metric":"icount","uncertain_reason":"BLOCK-MEASUREMENT-OVERHEAD"}}
```

Comment:
While it's not a big issue in this case, I generally prefer new test cases instead of modifying existing tests. Otherwise it's easy to lose test coverage accidentally.

The way this test is structured, it is particularly easy to add new cases. That is, add a line in `input_a` and `input_b` for each new test case. It's okay if it's all other fields are copy-paste from an existing test.
Then update the snapshot using `cargo insta test -p estimator-warehouse` and `cargo insta review`.

Reply:
Oh, I didn't mean a completely new test function. Sorry, I was not clear enough before what I mean by test case. But the test function `test_check_command` already has data to test multiple cases and now you copy-pasted all of them. This is probably too redundant and we are ending up with a lot of snapshots this way.

I meant that we can simply add a new data entry to test one more case.

Basically, instead of changing this
```
{"computed_in":{"nanos":119,"secs":46},"name":"LogBase","result":{"gas":6000000000.0,"instructions":48000.0,"io_r_bytes":0.0,"io_w_bytes":0.0,"metric":"icount","uncertain_reason":null}}
```
to this
```
{"computed_in":{"nanos":119,"secs":46},"name":"LogBase","result":{"gas":6000000000.0,"instructions":48000.0,"io_r_bytes":0.0,"io_w_bytes":0.0,"metric":"icount","uncertain_reason":"NEGATIVE-COST"}}
```
I was thinking about this
```
{"computed_in":{"nanos":119,"secs":46},"name":"LogBase","result":{"gas":6000000000.0,"instructions":48000.0,"io_r_bytes":0.0,"io_w_bytes":0.0,"metric":"icount","uncertain_reason":null}}
{"computed_in":{"nanos":119,"secs":46},"name":"WasmInstruction","result":{"gas":6000000000.0,"instructions":48000.0,"io_r_bytes":0.0,"io_w_bytes":0.0,"metric":"icount","uncertain_reason":"NEGATIVE-COST"}}
```

Reply:
I see. I did misunderstood your suggestion. I fixed it now. Please let me know if its okay.

Reply:
Yes look perfect, thanks!